game/scripts/npc/abilities/templar_assassin_refraction.txt

```diff
@@ -42,8 +42,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "20 40 60 100 200 300" // OAA
-      }
+        "bonus_damage"                                    "20 40 60 80 200 300"
```

Comment:
You removed } here?

Reply:
Yeah you did. That's why Travis is complaining.

Reply:
my bad lol